### PR TITLE
fixed/improved URI Admin page

### PR DESCRIPTION
**Pull request by camhashemi**
serializing targetArticle in UrlController, improving formatting in uri.scala.html, refactoring

### CODE REVIEW

**Comment by camhashemi** [comment]
@leogrim e.g. https://admin.kifi.com/admin/uri/153423. Let me know what you'd like to add/change. Normalization format will be fixed in the next commit and I'll remove admin/scraped once you give the thumbs up.

File: kifi-backend/modules/shoebox/app/views/admin/uri.scala.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @(uri: com.keepit.model.NormalizedURI, articleInfos: Set[com.keepit.rover.model.ArticleInfo], aggregateContent: com.keepit.rover.article.content.ArticleContentExtractor)(implicit request: com.keepit.common.controller.UserRequest[_])
 
-@admin(uri.title.getOrElse("Title Not Found")) {
+@admin(aggregateContent.title.getOrElse("Title Not Found")) {
```
Comment (leogrim) [inline kifi-backend/modules/shoebox/app/views/admin/uri.scala.html:3]: Can we just leave it empty? This page is about inspecting our data, so we want to return it as raw as possible.

**Comment by leogrim** [comment]
:+1:

File: kifi-backend/modules/shoebox/app/views/admin/uri.scala.html (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/views/admin/uri.scala.html:3]: I can do that. What about the URI id? it's not on the page, only in the page URL.

**Pull request merged**